pages/plugins.vue

```diff
@@ -17,7 +17,7 @@ export default {
   },
   head() {
     return {
-      title: "hapi.dev - Plugins",
+      title: "Plugins - hapi.dev",
       head() {
         return {
           title: "hapi.js - " + this.$route.params.family,
```

Comment:
@romMidnight would you mind deleting the unneeded nesting here but keeping the `title: "Plugins - hapi.dev",` and meta attribute, please?

Reply:
just did it in https://github.com/romMidnight/hapi.dev/commit/1b9fe666a48053c1e4bdf61f4431b5f8c16bf82f